umm-spec-lib/src/cmr/umm_spec/test/echo10_expected_conversion.clj

```diff
@@ -33,15 +33,18 @@
       (catch NumberFormatException e))))
 
 (defn- expected-file-dist-info
-  "Created expected FileDistributionInformation for ArchiveAndDistributionInformation map."
-  [file-dist-info]
-  (when-let [file-dist-info (first file-dist-info)]
-    (-> file-dist-info
-        (select-keys [:Fees :Format :FormatType])
-        (assoc :FormatType "Native")
-        (update :Fees echo10-expected-fees)
-        umm-c/map->FileDistributionInformationType
-        vector)))
+  "Created expected FileDistributionInformation for ArchiveAndDistributionInformation map.
+   In ECHO 10 there is only 1 price (fees) but there can be many data formats. The first fee
+   applies to all data formats."
+  [file-dist-infos]
+  (when file-dist-infos
+   (let [first-price (echo10-expected-fees (get (select-keys (first file-dist-infos) [:Fees]) :Fees))]
+    (for [file-dist-info file-dist-infos]
+      (-> file-dist-info
+          (select-keys [:Format :FormatType])
```

Comment:
Do we need select-keys here?  Do the other keys need to be stripped out?

Reply:
yes, because I need the map, but I don't need :FormatType anymore, so I have removed it.